emrun.py

```diff
@@ -350,15 +350,20 @@ def is_browser_process_alive():
   if browser_process and browser_process.poll() is None:
     return True
 
-  if current_browser_process_pids:
+  if current_browser_processes:
     try:
       import psutil
-      for p in current_browser_process_pids:
+      for p in current_browser_processes:
         if psutil.pid_exists(p['pid']):
           return True
     except Exception:
       # Fail gracefully if psutil not available
       return True
+  else:
```

Comment:
I think this makes the final return False unreachable.  How about doing this above:

```
if current_browser_processes:
   return page_exit_code is None
```

Then you don't need the else case?

Reply:
Revised this code in a slightly different way to make it more readable.